src/main/java/org/radarcns/service/DataSetService.java

```diff
@@ -220,12 +207,12 @@ public Dataset getAllRecordsInWindow(String projectName, String subjectId,
 
         TimeFrame timeFrame = new TimeFrame(start, end);
 
+        SourceDataMongoWrapper sensorDao = this.sourceCatalog.getSourceDataWrapper(sourceDataName);
```

Comment:
Minor remark: perhaps `sensorDao` can be `sourceDataWrapper` or something similar.